bridge/qb/client/functions.lua

```diff
@@ -183,13 +183,23 @@ end
 
 ---@deprecated use lib.getClosestVehicle from ox_lib
 functions.GetClosestVehicle = function(coords)
+    if coords then
+        coords = type(coords) == 'table' and vec3(coords.x, coords.y, coords.z) or coords
+    else
+        coords = GetEntityCoords(cache.ped)
+    end
```

Comment:
```suggestion
    coords = type(coords) == 'table' and vec3(coords.x, coords.y, coords.z) or coords or GetEntityCoords(cache.ped)
```
This part can be reduced.